### PR TITLE
Fix #1 Fix #2 Fix #3 Fix #4 Fix #5 Fix #6 Fix #7 Fix #8

### LINKED ISSUE #1: data preprocessing
- [x] Collect data
- [x] Organize training set and test set 
- [x] cleaning the data
- [x] data split

### LINKED ISSUE #2: Update project documentation
- [x] README
- [x] contributing guide
- [x] code of conduct

### LINKED ISSUE #3: [COURSE] - Apply tests with  Self organizing Maps

### LINKED ISSUE #4: Train model

### LINKED ISSUE #5: analyze results

### LINKED ISSUE #6: Improve the model
Improve the network to compare with the result of 95% already achieved with the result of the teacher

### LINKED ISSUE #7: Show and explain me the solution
- [x] introduction (explain the project and objectives)
- [x] Technical Explanation 
- [ ] Lessons Learned (gathering data take time, hours of training, CNN)
- [ ] Conclusion and Future Works (Create CI/CD for releases, and define a pipeline)

### LINKED ISSUE #8: Save model to disk

### PR DESCRIPTION
**Pull request by eliseuegewarth**

### CODE REVIEW

**Review by alvesisaque — APPROVED** [review]
it's all ok.
